pro/src/screens/OfferEducational/OfferEducationalForm/FormOfferType/__specs__/FormOfferType.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { screen } from '@testing-library/react'
+import { Formik } from 'formik'
+
+import { renderWithProviders } from 'utils/renderWithProviders'
+
+import FormOfferType, { FormTypeProps } from '../FormOfferType'
+
+const formTypeProps: FormTypeProps = {
+  categories: [],
+  disableForm: false,
+  domainsOptions: [],
+  nationalPrograms: [],
+  subCategories: [],
+}
+
+function renderFormOfferType(props: FormTypeProps) {
+  return renderWithProviders(
+    <Formik initialValues={{ title: '', domains: [] }} onSubmit={vi.fn()}>
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <FormOfferType {...props} />
+        </form>
+      )}
+    </Formik>
+  )
+}
+
+describe('FormOfferType', () => {
+  it('should render the form description', async () => {
+    renderFormOfferType(formTypeProps)
+    expect(
+      await screen.findByText(
```

Comment:
ça fonctionne pas avec un getBy tout simple ? (et du coup enlever le async)

Reply:
Avec le `getBy` j'ai l'erreur jest `Unknown Error: Warning: An update to %s inside a test was not wrapped in act(...).` à cause du `useEffect` qui contient un `setFieldValue` qui est awaited. Je vois 3 solutions :
- ne pas await le `setFieldValue` (ce que je fais d'habitude quand j'ajoute du code, mais ça obligé aussi a ajouter un commentaire eslint pas méga beau -- est-ce qu'il y a une directive là dessus ?)
- Utiliser le `findByText` qui attend juste ce qu'il faut pour que le useEffect soit terminé avant de finir le test
- Utiliser un `waitFor` qui vérifie que l'effet du `setFieldValue` a bien eu lieu avant de faire le `expect(     screen.getByText(...` mais c'est toujours une fonction asynchrone